server/server.go

```diff
@@ -60,6 +62,17 @@ func StartServer() {
 		}
 	}()
 
+	// Add profiling flag to enable profiling routes.
+	if os.Getenv("PPROF_ENABLED") != "" {
+		// pprof attaches routes to default serve mux
+		// host:6061/debug/pprof/
+		go func() {
+			if err := http.ListenAndServe(":6061", http.DefaultServeMux); err != nil {
```

Comment:
Looks fine. You are explicitly enabling pprof based on a flag. I assume pprof won't appear on port 8192 because you've set a specific `Handler` right?

Hmm I'm not sure this part of the rule message makes sense:

>  Instead, use import "net/http/pprof"